### PR TITLE
[vim/fr-fr] New translation

### PR DESCRIPTION
**Pull request by napnac**
- [X] PR touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Seriously, look at it now. Watch for quotes and double-check field names.

Hope this helps ! ;)

### CODE REVIEW

File: fr-fr/vim.html.markdown [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+translators:
+    - ["Thibault", "https://github.com/napnac"]
+lang: fr-fr
+---
+
+
+[Vim](www.vim.org)
+(Vi IMproved) est le clone le plus populaire de l'éditeur de texte vi sous Unix.
+Vim est un éditeur de texte omniprésent sur les systèmes de type Unix, et a pour
+objectif la rapidité ainsi que l'augmentation de la productivité. Ce dernier a
```
Comment (vendethiel) [inline fr-fr/vim.html.markdown]: Ce dernier => Il

File: fr-fr/vim.html.markdown [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+## Navigation basique avec Vim
+
+```
+    vim <ficher>     # Ouvre <fichier> avec vim
+    :q               # Quitte vim
+    :w               # Sauvegarde le fichier actuel
+    :wq              # Sauvegarde le fichier actuel et quitte vim
+    :q!              # Quitte vim sans sauvegarder
+                     # ! *force* l'exécution de :q, ce qui par conséquent 
+                        oblige vim à quitter sans sauvegarder
```
Comment (vendethiel) [inline fr-fr/vim.html.markdown]: manque un `#`

File: fr-fr/vim.html.markdown [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    # Se déplacer vers un caractère
+
+    f<caractère>     # Se déplace en avant jusqu'à <caractère>
+    t<caractère>     # Se déplace en avant juste avant <caractère>
+
+    # Par exemple
+    f<               # Se déplace en avant jusqu'à <
+    t<               # Se déplace en avant juste avant <
+    
+    # Se déplacer vers un mot
```
Comment (vendethiel) [inline fr-fr/vim.html.markdown]: pas "vers". "dans" ou "à travers d'un"

File: fr-fr/vim.html.markdown [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+    y                # Copie le texte sélectionné
+    yy               # Copie la ligne actuelle
+    d                # Supprime ce qui est sélectionné
+    dd               # Supprime la ligne actuelle
+    p                # Colle après le curseur
+    P                # Colle avant le curseur
+    x                # Supprime le caractère sous le curseur
+```
+
+## La 'Grammaire' de Vim
```
Comment (vendethiel) [inline fr-fr/vim.html.markdown]: ", pas '

**Comment by vendethiel** [comment]
LGTM. Je merge ?

**Comment by napnac** [comment]
Are you waiting on some other changes?

**Comment by vendethiel** [comment]
Nope! Merci !

**Pull request merged**